Program.cs

```diff
@@ -0,0 +1,377 @@
+﻿using AirportTicketBooking;
+using static AirportTicketBooking.Flight;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+Console.WriteLine("Welcome to Airport Ticket Booking system");
+Console.Write("Enter the path of the CSV file: ");
+string filePath = Console.ReadLine();
+Booking bookingSystem = new Booking(filePath);
+List<Passenger> passengers = new List<Passenger>();
+List<Manager> managers = new List<Manager>();
+bool exitSystem = false;
+while (!exitSystem)
+{
+Console.WriteLine("\nWhich type of user are you?");
+Console.WriteLine("1. Passenger");
+Console.WriteLine("2. Manager");
+Console.WriteLine("3. Exit");
+int choice = int.Parse(Console.ReadLine());
+
+switch (choice)
+{
+case 1:
+PassengerMenu();
+break;
+
+case 2:
+ManagerMenu();
+break;
+
+case 3:
+exitSystem = true;
+break;
+
+default:
+Console.WriteLine("Invalid choice. Please try again.");
+break;
+}
+}
+
+void ManagerMenu()
+{
+    Console.WriteLine("Please enter your details:");
+    Console.Write("Name: ");
+    string name = Console.ReadLine();
+    Console.Write("Email: ");
+    string email = Console.ReadLine();
+    Console.Write("Password: ");
+    string password = Console.ReadLine();
+    Manager manager = null;
+    Manager GetOrCreateManager(string name, string email, string password)
+    {
+        Manager existingManager = managers.FirstOrDefault(p => p.Name == name && p.Email == email && p.CheckPassword(password));
+        if (existingManager != null)
+        {
+            Console.WriteLine($"Welcome back, {existingManager.Name}!");
+            return existingManager;
+        }
+        else
+        {
+            Manager newManager = new Manager
+            {
+                Name = name,
+                Email = email,
+            };
+            newManager.SetPassword(password);
+            managers.Add(newManager);
+            Console.WriteLine($"Welcome, {newManager.Name}!");
+            return newManager;
+        }
+    }
```

Comment:
why do we have this code in the middle of the logic?
what about having a ManagerRepository that has the CRUD operations?

Reply:
Also, this violates the SRP you will take it later in solid principles, so we usually return the manager id from creation method, then we retrieve the manager by that id